com.unity.multiplayer.transport.utp/Runtime/UTPTransport.cs

```diff
@@ -260,14 +254,14 @@ public void SendToClient(NativeArray<byte> packet, ulong clientId, int index)
         }
     }
 
-    public override unsafe void Send(ulong clientId, ArraySegment<byte> data, NetworkChannel networkChannel)
+    public override unsafe void Send(ulong clientId, ArraySegment<byte> data, NetworkDelivery networkDelivery)
     {
         var pipelineIndex = 0;
 
         GetUTPConnectionDetails(clientId, out uint peerId);
 
         var writer = new DataStreamWriter(data.Count + 1 + 4, Allocator.Temp);
-        writer.WriteByte((byte)networkChannel);
+        writer.WriteByte((byte)networkDelivery);
```

Comment:
So, do we actually need to send the network delivery?  I was assuming before we sent the channel so that we could on the receiving end route to the right channel, but if we don't do that... (or maybe I don't understand what this is for)

Reply:
no, you're right. we don't need this and channel ID at all. I wanted to keep better UTPTransport implementation for later, see my comments above.

Reply:
hey @andrews-unity, if you want me to kill this in this PR, I can do that here even before we make UTPTransport great again :)

Reply:
Kill it

Reply:
Double Kill it!

Reply:
Duh, we have to also delete the reading of it in `Execute` and `ServerUpdateJob`